cmd/extender/extender.go

```diff
@@ -42,7 +43,8 @@ type extendCmd struct {
 
 func (e *extendCmd) DefineFlags() {
 	cmd.FlagAppDir(&e.appDir)
-	cmd.FlagCacheDir(&e.cacheDir)
+	cmd.FlagCacheDir(&e.cacheDir) // TODO: will this ever be used by kaniko? Should we remove it?
```

Comment:
Only if we want platforms to dump build/run image bases as tarballs so kaniko doesn't have to pull them in during extension runs. Could be useful as a readonly volume if the platform allows only certain base images.